build_tools/python/benchmark_suites/iree/benchmark_collections.py

```diff
@@ -51,7 +53,10 @@ def generate_benchmarks(
             # Enable zip polyglot to provide component sizes.
             "--iree-vm-emit-polyglot-zip=true",
             # Disable debug symbols to provide correct component sizes.
-            "--iree-llvmcpu-debug-symbols=false"
+            "--iree-llvmcpu-debug-symbols=false",
+            # Dump scheduling statistics
+            "--iree-scheduling-dump-statistics-format=json",
```

Comment:
We're also benchmarking compiler latency here. Is this going to significantly affect that?

Reply:
Tested to compile ClipTextSeqLen64PT for x86_64
```sh
iree-compile ../iree-build/e2e_test_artifacts/model_9a9515c7-cb68-4c34-b1d2-0e8c0a3620b8_ClipTextSeqLen64PT.mlir -o ../iree-build/e2e_test_artifacts/iree_ClipTextSeqLen64PT_module_f13d9b8b55decd1684c262d230ad42c95c5def21d9ee605ebc656181d1c54a0a/module.vmfb --iree-hal-target-backends=llvm-cpu --iree-input-type=none --iree-llvmcpu-target-triple=x86_64-unknown-linux-gnu --iree-llvmcpu-target-cpu=cascadelake --iree-vm-emit-polyglot-zip=true --iree-llvmcpu-debug-symbols=false --iree-scheduling-dump-statistics-format=json --iree-scheduling-dump-statistics-file=../iree-build/e2e_test_artifacts/iree_ClipTextSeqLen64PT_module_f13d9b8b55decd1684c262d230ad42c95c5def21d9ee605ebc656181d1c54a0a/scheduling_stats.json --mlir-timing
```
The time seems to be negligible
```
===-------------------------------------------------------------------------===
                         ... Execution time report ...
===-------------------------------------------------------------------------===
  Total Execution Time: 13.2729 seconds   
    
  ----User Time----  ----Wall Time----  ----Name----
...
0.0128 (  0.0%)    0.0128 (  0.1%)  DumpStatistics
...
```